console/EAP7-752_Support_kill_and_destroy_operations_for_server_groups.adoc

```diff
@@ -0,0 +1,43 @@
+= Support kill and destroy operations for server-groups
+:author:            Harald Pehl
+:email:             hpehl@redhat.com
+:toc:               left
+:icons:             font
+:idprefix:
+:idseparator:       -
+:issue-base-url:    https://issues.jboss.org/browse/
+
+== Overview
+
+Expose runtime data of deployed EJBs
+
+== Issue Metadata
+
+=== Issue
+
+* https://issues.jboss.org/browse/EAP7-749[EAP7-749]
+
+=== Dev Contacts
+
+* mailto:hpehl@redhat.com[Harald Pehl]
+
+=== QE Contacts
+
+* mailto:pjelinek@redhat.com[Pavel Jelinek]
+
+=== Affected Projects or Components
+
+* WildFly
+* HAL
+
+== Requirements
+
+Deployed EJBs expose their runtime data and statistics under the runtime top level category.
```

Comment:
Please update text for EAP7-752. This is copy-pasted from https://github.com/wildfly/wildfly-proposals/pull/91 .